frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -307,6 +314,10 @@ export function DashboardItem({
                         </div>
                     )}
                 </div>
+                {item.description && (
+                    <div style={{ padding: '0 16px', marginBottom: 16, fontSize: 12 }}>{item.description}</div>
+                )}
+
```

Comment:
This really should allow for editing now IMO, as the descriptions will become irrelevant and confusing when the user updates the dashboard item with some other insight than the initial one.

Reply:
I agree, I've opened #2075 to tackle this as an improvement to be able to get this in before the code freeze in the meantime.